no-computer.md

```diff
@@ -11,34 +11,43 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Explaining game rules can be pretty hard. A lot of people strugle to explain game rules to each other and it only gets harder when explaining them to something like a computer. By creating a DSL for game rules, we can standardize game rule formate and maybe even help game AI makers in the process if the syntax is rigorous enough.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Describing games and the rules to play them does not require a lot of different terminology for many board games. Thus, a limited-scope DSL could fit this purpose. The DSL would standardize how game rules are described and enourage clarity so that users can understand them better.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+What would be really really cool is if the rules are specific enough that you could automatically generate a computer version of the game and/or an AI for the game. This idea was inspired by my idea to write up a board game (like Monopoly) for the no computer project and by a [previous final project](https://github.com/AriHC/RuleBook/blob/master/documents/final.md) from DSLs.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Standardized game rule description language.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would simply consume the "programs" in the language to learn about the game described. Readability will be key as this would technically be aimed at people, but it would be acceptable to make it more strict so that computers can parse it as well.
```

Comment:
This is interesting since this does seem separated from a computer, but it seems like this DSL would benefit a lot from being able to interface with a computer. As a project idea it has merit in both ways, perhaps.